### ISSUE TITLE
[Question]: Does the RAGFLOW support: "Handling Concurrent Sessions and Chronological Messaging"?

### ISSUE THREAD

**Issue by phamkhactu** [infiniflow/ragflow#3609]
### Describe your problem

Thanks for your work,

I want to know Does the model support  "Handling Concurrent Sessions and Chronological Messaging"?. If yes, Could you share with me the architecture about it.

I have read code, but I did not extractly find the architecture.
I'm look forward to hearing from you as soon as posible.

**Comment by KevinHuSh** [comment]
Do you mean flask do not support concurrent requests?

**Comment by phamkhactu** [comment]
No, I don't. Flask support concurrent requests, but with multiple requests such as 1 millions or more, can ragflow works well? I reads some blog which using redis, queue... etc. How about you?
